PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -258,7 +258,7 @@ DECLARE_SOA_TABLE(ReducedMuons, "AOD", "RTMUON", //!
 DECLARE_SOA_TABLE(ReducedMuonsExtra, "AOD", "RTMUONEXTRA", //!
                   fwdtrack::NClusters, fwdtrack::PDca, fwdtrack::RAtAbsorberEnd,
                   fwdtrack::Chi2, fwdtrack::Chi2MatchMCHMID, fwdtrack::Chi2MatchMCHMFT,
-                  fwdtrack::MatchScoreMCHMFT, fwdtrack::MFTTrackId, fwdtrack::MCHTrackId,
+                  fwdtrack::MatchScoreMCHMFT, fwdtrack::MFTTrackId, reducedmuon::MCHTrackId,
```

Comment:
Since its useless, and actually problematic when merging, please remove the MFTTrackId, as we discussed during the last meeting.